InferenceSystem/src/LiveInferenceOrchestrator.py

```diff
@@ -101,7 +103,7 @@ def populate_metadata_json(
 	# logger to app insights
 	con_string = os.getenv('INFERENCESYSTEM_APPINSIGHTS_CONNECTION_STRING')
 	logger = logging.getLogger(__name__)
-	if appInsightsKey is not None:
+	if con_string is not None:
```

Comment:
Can we rename the variable to be more specific? (such as app_insights_connection_string)

Reply:
renamed